lemur/common/validators.py

```diff
@@ -41,22 +42,33 @@ def private_key(key):
         raise ValidationError('Private key presented is not valid.')
 
 
+def common_name(value):
+    """If the common name could be a domain name, apply domain validation rules."""
+    # Common name could be a domain name, or a human-readable name of the subject (often used in CA names or client
+    # certificates). As a simple heuristic, we assume that human-readable names always include a space.
+    # However, to avoid confusion for humans, we also don't count spaces at the beginning or end of the string.
```

Comment:
I'm a bit confused by this, are you saying that all of your users that enter a common name happen to have a space in them? We often see myCommonName in client certificates (java programmers). So I'm not sure this would work for us. Could this heuristic be changed to look for '.' in the common name and then attempt to validate? 

```
if '.' in value:
    return sensitive_domain(value)
```
I think there is a strong assumption that if you are entering periods into a common name an FQDN is expected.